### PR TITLE
OSDOCS-483: Included GCE Persistent Disk details.

### PR DESCRIPTION
**Pull request by huffmanca**
Included GCE Persistent disk details.

This is for OS 4.2.

### CODE REVIEW

**Comment by huffmanca** [comment]
@openshift/team-documentation ,

Is it possible to review this one?

File: storage/persistent-storage/persistent-storage-gce.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+.Additional references
+
+link:https://cloud.google.com/compute/docs/disks/[GCE Persistent Disk]
```
Comment (bergerhoffer) [inline storage/persistent-storage/persistent-storage-gce.adoc]: Should be a bullet.

File: modules/dynamic-provisioning-available-plugins.adoc [hunk 2]
```diff
@@ -27,11 +27,11 @@ configured provider's API to create new storage resources:
...
-//|GCE Persistent Disk (gcePD)
-//|`kubernetes.io/gce-pd`
-//|In multi-zone configurations, it is advisable to run one {product-title} 
-//cluster per GCE project to avoid PVs from getting created in zones where 
-//no node from current cluster exists.
+|GCE Persistent Disk (gcePD)
+|`kubernetes.io/gce-pd`
+|In multi-zone configurations, it is advisable to run one {product-title}
+cluster per GCE project to avoid PVs from getting created in zones where
+no node from current cluster exists.
```
Comment (bergerhoffer) [inline modules/dynamic-provisioning-available-plugins.adoc]: from current cluster -> from the current cluster?

File: storage/persistent-storage/persistent-storage-gce.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+GCE Persistent Disk volumes can be provisioned dynamically.
+
+Persistent volumes are not bound to a single project or namespace;
+they can be shared across the {product-title} cluster.
+Persistent volume claims are specific to a project or namespace and can be
+requested by users.
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
```
Comment (bergerhoffer) [inline storage/persistent-storage/persistent-storage-gce.adoc]: "High availability" here shouldn't be hyphenated.

**Review by bergerhoffer — COMMENTED** [review]
Just a few minor things, otherwise LGTM!

File: storage/persistent-storage/persistent-storage-gce.adoc (hunk 1 above)
Comment (huffmanca) [reply storage/persistent-storage/persistent-storage-gce.adoc]: Changed this to a bullet.

File: modules/dynamic-provisioning-available-plugins.adoc (hunk 2 above)
Comment (huffmanca) [reply modules/dynamic-provisioning-available-plugins.adoc]: Adjusted this sentence to make it read a bit cleaner.

File: storage/persistent-storage/persistent-storage-gce.adoc (hunk 3 above)
Comment (huffmanca) [reply storage/persistent-storage/persistent-storage-gce.adoc]: Removed the hyphen in this word.

**Comment by vikram-redhat** [comment]
@liangxia could you please review?

**Comment by liangxia** [comment]
LGTM (from QE)

**Comment by vikram-redhat** [comment]
@huffmanca ok to merge?

**Pull request merged**